modules/openapi-generator/src/main/resources/Java/libraries/okhttp-gson/anyof_model.mustache

```diff
@@ -210,13 +210,11 @@ public class {{classname}} extends AbstractOpenApiSchema{{#vendorExtensions.x-im
         super("anyOf", {{#isNullable}}Boolean.TRUE{{/isNullable}}{{^isNullable}}Boolean.FALSE{{/isNullable}});
     }
 
-    {{#anyOf}}
-    public {{classname}}({{{.}}} o) {
+    public {{classname}}(Object o) {
```

Comment:
fyi. this is consistent with the constructor implementation in oneOf

Reply:
@Bethibande what about applying a fix to avoid duplicated constructor instead of using `Object` as now the constructor allow any type to be provided (no type validation)?